e2e/pnpm_lockfiles/v101/snapshots/lodash-4.17.21_tgz_defs.bzl

```diff
@@ -12,7 +12,7 @@ load("@aspect_rules_js//npm/private:npm_import.bzl",
 PACKAGE = "lodash"
 VERSION = "file:../vendored/lodash-4.17.21.tgz"
 _ROOT_PACKAGE = "<LOCKVERSION>"
-_PACKAGE_STORE_NAME = "lodash@0.0.0"
```

Comment:
Notice how the package store name here and the `VERSION` above were different. Now they align, and the store name is the exact same as the pnpm store name (including `file+..+` chars).